src/server/QXmppIncomingClient.h

```diff
@@ -41,7 +41,8 @@ class QXMPP_EXPORT QXmppIncomingClient : public QXmppStream
     Q_OBJECT
 
 public:
-    QXmppIncomingClient(QSslSocket *socket, const QString &domain, QObject *parent = 0);
+    QXmppIncomingClient(QSslSocket *socket, const QString &domain, bool csiEnabled = false,
+                        QObject *parent = 0);
```

Comment:
Warning: This change breaks compatibility. Alternative were another constructor and defaulting `csiEnabled` to false with the current one.